sim/battle.ts

```diff
@@ -3123,7 +3129,7 @@ export class Battle extends Dex.ModdedDex {
 	}
 
 	getDebugLog() {
-		return this.log.join('\n').replace(/\|split\n.*\n.*\n.*\n/g, '');
+		return this.log.join('\n').replace(/\|split|.*\n.*\n.*\n/g, '');
```

Comment:
This is again incorrect, and is again something best handled with the function I pasted in comments.